pkg/operator/ceph/csi/csi.go

```diff
@@ -31,7 +31,7 @@ import (
 	"k8s.io/apimachinery/pkg/version"
 )
 
-func (r *ReconcileCSI) validateAndConfigureDrivers(serverVersion *version.Info, ownerInfo *k8sutil.OwnerInfo) error {
+func (r *ReconcileCSI) validateAndConfigureDrivers(serverVersion *version.Info, ownerInfo *k8sutil.OwnerInfo, csiRootPath string) error {
```

Comment:
you can add this here https://github.com/rook/rook/blob/756628c88569b3259f7a27b7cdcad00868168728/pkg/operator/ceph/csi/spec.go#L190 we don't need to pass it as argument to the function

Reply:
Thanks,
I missed looking the two references of `crushRoot` are in the same package `csi`, so easily a variable would work